libvmi/events.c

```diff
@@ -755,6 +755,48 @@ vmi_set_mem_event(
     return VMI_FAILURE;
 }
 
+status_t
+vmi_set_mem_range_event(
```

Comment:
I think the name `vmi_set_mem_event_range` would fit the API better.

Reply:
I would argue the opposite, since every other such API has `_event` suffix. But semantically setting "event" for "mem range" and setting "mem event" for a "range" should be same. So... if you insist, I'll change it.

Should I change it only in VMI interface, or everywhere? (eg. driver_wrapper, xen_events, ...)

Reply:
If you change it it should be consistent acrosss

Reply:
Renamed + fixed the scan-build issue